data/two_hex_overlap.mesh

```diff
@@ -0,0 +1,55 @@
+MFEM mesh v1.0
+
+#
+# MFEM Geometry Types (see mesh/geom.hpp):
+#
```

Comment:
Maybe add a comment on how much they overlap and general dimension of the hex cubes?

Reply:
Done